src/icons/GoogleIcon.tsx

```diff
@@ -1,15 +1,15 @@
 const GoogleIcon = (props:any) => {
   return (
     <svg
-      width="18"
-      height="18"
+      width="15"
```

Comment:
why is this one 15 and the others 16? Is there any specific reason?

Reply:
yes, I tried 15-16-18 for each one, from my point of view, it seems more likely to resemble the original ones in these sizes.